conf/modules.config

```
@@ -255,6 +252,20 @@ process {
         ]
     }
 
+    withName: BWA_MEM {
+        ext.args = { "-k ${params.mapping_bwamem_k} -r ${params.mapping_bwamem_r}" }
+        publishDir = [
+            enabled: false
+        ]
+    }
+
+    withName: ".*MAP:SAMTOOLS_INDEX_MEM" {
```

I think we can even remove the ugly warning now as the name is unique
```suggestion
    withName: "SAMTOOLS_INDEX_MEM" {
```